lib/rules/grapheme-string-literal.ts

```diff
@@ -0,0 +1,49 @@
+import type { RegExpVisitor } from "@eslint-community/regexpp/visitor"
+import type { RegExpContext } from "../utils"
+import { createRule, defineRegexpVisitor } from "../utils"
+
+const segmenter = new Intl.Segmenter()
+
+export default createRule("grapheme-string-literal", {
+    meta: {
+        docs: {
+            description: "enforce single grapheme in string literal",
+            category: "Best Practices",
```

Comment:
I'm not sure whether this should be a best practice. The practice this rule enforces is good, because it brings regexes into a canonical form (e.g. don't use `[\q{foo|bar}]`, use `(?:foo|bar)`), but that's more of a stylistic thing IMO.

Reply:
I agree with you 👍  I will categorize it under "Stylistic Issues".